src/DonationForms/resources/styles/design-settings/image.scss

```diff
@@ -216,3 +218,28 @@
     }
 }
 
+/* Two-Panel */
+.givewp-donation-form > .givewp-layouts-header {
+    .givewp-layouts-header__templates-ms {
+        padding-bottom: 2rem;
+
+        &--background {
+            .givewp-layouts-header__content {
+                position: static;
+                margin-bottom: 0;
+            }
+
+            .givewp-layouts-headerImage {
+
+                img {
+                    position: absolute;
+                    left: 0;
+                    bottom: 0;
+                    height: 100%;
+                    min-height: 100%;
+                    border-radius: var(--givewp-rounded-8);
+                }
+            }
```

Comment:
@JoshuaHungDinh So the `.givewp-layouts-headerImage` isn't applied to the actual `img` tag? I'm guessing not because it depends on the image position?

Reply:
It's not being applied to the image tag because of the use of `withTemplateWrapper`. The image position is a dynamic classname that wraps around the header template so that we can easily create the ideal html structure for each position.